lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -5,7 +5,13 @@
 from django.utils.translation import pgettext
 from openedx.core.djangolib.markup import HTML, Text
 %>
-% if not section_data['access']['eshe_instructor'] or section_data['access']['explicit_staff']:
+
+<%
+  eshe_instructor_or_teaching_assistant_access = section_data['access']['eshe_instructor'] or section_data['access']['teaching_assistant']
+  membership_section_visible = not eshe_instructor_or_teaching_assistant_access or section_data['access']['explicit_staff']
+%>
+
+% if membership_section_visible:
```

Comment:
@0x29a Can you please explain why checking for only `explicit_staff` won't work here?